backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions/Docs_Sales_Order_Details.sql

```diff
@@ -124,6 +124,7 @@ FROM C_OrderLine ol
 WHERE ol.C_Order_ID = p_record_id
   AND ol.isActive = 'Y'
   AND (COALESCE(pc.M_Product_Category_ID, -1) != getSysConfigAsNumeric('PackingMaterialProductCategoryID', ol.AD_Client_ID, ol.AD_Org_ID))
+  AND ol.isHideWhenPrinting != 'Y'
```

Comment:
what about taxes?
what if line net amount / tax amt is not zero?

are were somewhere guarding against that case?

Reply:
In this case, it's about hiding products in the same compensation group, and in the order, only showing the _main_ compensation group product. The other usages I found are related to quotation, and seem to match the expectations here.

Reply:
if the isHideWhenPrinting  is not introduced in this issue, then let's flag is conversation as resolved

Reply:
As agreed with Torby, we're going to create a data advisory for such cases